ansible/roles/common/files/ssh-keys/shep.pub

```diff
@@ -1,2 +1,3 @@
 ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCiqtpfx4sjF5qVLP11O/y/xOpgsyPVsw56GzDfDidOZpBnXg92Fl2gK/7jNctL7YIvyr93z2O5W74Yz1GkU6Nudvc3X7wqzbt/w9lKK+xbi00i3Z+2MLQ4p5iAHqdLboYJJb2S0SUfeBrxVy2dJdTw8lV2BiWwJuckBHm86/fCJbKv4ZEmf0YuyfB5aTeiz3lRTeJAlIlbBG7EPggj+XVVPD3s466X15Pvi6fBMrM8586VNLV0xf9hRHjNqk6VSzZbvl0i0pF671GH/SFrlGLebs+0M9cThR71UPzOT4XOP5jawKJ192jjKvAxPkt0hIj3I8JXGzFO7+fP+0zzOKoL
 ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC3rxbaypWWXTNQV3VLgMLMPIi3MiMXxUfSMDrAcfYVx6PrA9Lzuph1ysx7aPdHwGdwQ3oCb+Y8lAubsdn8Re2HU1lQkobk3ucapp3OGNqUmXUuueMfYShz5sb42AXMMLrRwr5igGBO47MTDKdGtpQs7vED+pyLKR2ibxHFC2euDUvwtrfrTFa0Cr2iAw9pnRGok33jReAPXZ5BwVyctbTKtDg85arO4ZC65IVkgsVXi9Vq8rE4CU+pW36JJ32PjQIkAQR1fQJUjkFwqRe8hZSNFYDEJ6AUDrfpomWHjc/8jt8Bhh8/h3KOrRn7rENT83yJbR+NUvDQc6xtiP1QuJ7I6lQzh094VdLpcK5rCUPWKHiKiCA2JeIyxthZjkB7j32xfYJMaCwjI8DdUAmudoM9kzapZGQFuD0xTys2tuMFKn5Z4oFu+2bQ1OZe2wsxmMbLEfhFuiSVVTaxQZd4ImWWzGPzGTsMRTdiSNRBhdP+dWbIHbO36Rj9PnD6uZnMq9IuMHU9/li9dW6l86hh6Qm1xESb8zAVk8vfxfLJpRJz5Oq2iyocuFzy/VoC9D3PPZmq3WBGJOjrtUO+xEl/EOkqH6EVBVI8jwLtOPVZVPPBCusX8d0MLb35OMNeUk6LqVxfXOPeQ2UBu7O0+wA3aRNXWVGFlaDJxqvSPYUrAkFocQ==
```

Comment:
I don't actually recognize this existing key. The first line is my old laptop — is this some internal infra key?

Reply:
No, shouldn't be. I would suggest removing it if we don't know what it is.

Reply:
Removed!